packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -597,6 +597,22 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setUpLocusMediaSharesListener();
     this.setUpLocusInfoMeetingInfoListener();
     this.setUpLocusInfoAssignHostListener();
+    this.setUpLocusInfoMediaInactiveListener();
+  }
+
+
+  /**
+   * Set up the locus info listener for meetings disconnected due to inactivity
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  setUpLocusInfoMediaInactiveListener() {
+    this.locusInfo.on(EVENTS.DISCONNECT_DUE_TO_INACTIVITY, (res) => {
+      // TODO: need to send metric for locus disconnect
+      LoggerProxy.logger.error(`Meeting:index#setUpLocusInfoMediaInactiveListener --> Meeting disconnected due to inactivity: ${res.reason}`);
+      this.reconnect();
```

Comment:
Is it safe to call this function yet? We removed its call from the ice disconnect logic until we had further testing. I'd rather mark this as a TODO for once the reconnect logic is solid.

Reply:
I am guessing there is a reconnect config for meetings plugin we can check that . If our testing goes good we can ask them to enabled the toggle with out a code changes